.dev/gather_benchmark_train_results.py

```diff
@@ -72,9 +72,9 @@ def parse_args():
                 print(f'log file error: {log_json_path}')
                 continue
 
-            differential_results = dict()
```

Comment:
$ python3 -m timeit "x = dict()"
5000000 loops, best of 5: 69.8 nsec per loop

Reply:
Thanks for you improving mmseg code efficiency